docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -343,27 +350,278 @@ you need to list explicitly all the non-bundle extras that you want to install.
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_ssh                    | pip install -e '.[editable_ssh]'                    | Adds all libraries needed by the ssh provider              |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_tableau                | pip install -e '.[editable_tableau]'                | Adds all libraries needed by the tableau provider          |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_tabular                | pip install -e '.[editable_tabular]'                | Adds all libraries needed by the tabular provider          |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_telegram               | pip install -e '.[editable_telegram]'               | Adds all libraries needed by the telegram provider         |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_trino                  | pip install -e '.[editable_trino]'                  | Adds all libraries needed by the trino provider            |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_vertica                | pip install -e '.[editable_vertica]'                | Adds all libraries needed by the vertica provider          |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_weaviate               | pip install -e '.[editable_weaviate]'               | Adds all libraries needed by the weaviate provider         |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_yandex                 | pip install -e '.[editable_yandex]'                 | Adds all libraries needed by the yandex provider           |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+| editable_zendesk                | pip install -e '.[editable_zendesk]'                | Adds all libraries needed by the zendesk provider          |
++---------------------------------+-----------------------------------------------------+------------------------------------------------------------+
+
+Doc extras
```

Comment:
As `devel` and `devel_all` is described in the chapter below I'd propose to switch order such that the "Doc extras" are below "Bundle devel extras" or merge both lists as they are actually belonging close together.

Reply:
I moved `docs` down. Actually I have some plans to remove `doc` extra eventually, I **think** It's not needed and holds us down a bit with CI image, and I think we should move  doc generation outside of our CI image - but I want to do it as next step, this is quite a bit more changes to `breeze` and other places, so I prefer to keep it separate.